outwatch/outwatch
# run benchmarks and tests in browser

- use colibri version 0.1.0
- Fix example in readme
- gh-actions build also checks out git tags
- Update scalatest to 3.2.10 (#530)
- Update sbt-mdoc to 2.2.24 (#533)
- Mergify: Scala 2.13.7
- Mergify: Match for major scala versions
- Update colibri, colibri-monix to 0.1.1 (#543)
- CI: Allow publishing pull requests
- CI publish: fix commit sha
- Update colibri, colibri-monix to 0.1.2 (#546)
- Update domtypes to 0.15.1 (#547)
- handle errors properly in dynamic content (IO/Observables) (#545)
- show latest maven release in documentation
- caching in github action
- only run website action on tag
- improve error style
- remove old contributing.md
- Update sbt to 1.5.6 (#550)
- Update sbt to 1.5.7 (#554)
- Update dependencies in readme
- Update sbt to 1.5.8 (#555)
- Update website texts (#556)
- Explicitly mention mdoc preview mechanisms (#557)
- Fix sbt deprecation warnings (#558)
- Remove obsolete CI strategies
- Explain RenderConfig in docs
- Remove js.UndefOr extractor
- Deprecate handler
- Add Felix Dietze to developers
- Use ScalaJS source mapping plugin
- Update sbt to 1.6.0 (#559)
- Update sbt to 1.6.1 (#560)
- Update scalajs-dom to 2.1.0 (#561)
- Update to Scala.js 1.8.0 (#563)
- Fix SJS 1.8 update
- update colibri version to 0.2.1
- Update colibri, colibri-jsdom, ... to 0.2.2 (#567)
- Fix npm github dependency (#568)
- Update silencer-lib, silencer-plugin to 1.7.8 (#569)
- add inner html
- Update scalatest to 3.2.11 (#572)
- Update yarn.lock
- Update template port
- Update colibri, colibri-jsdom, ... to 0.2.3 (#574)
- Update sbt to 1.6.2 (#575)
- Update colibri, colibri-jsdom, ... to 0.2.5 (#576)
- Update sbt-scalajs, scalajs-compiler, ... to 1.9.0 (#580)
- accum for style attributes
- remove wrong accum method for concrete styles
- render cats.data.NonEmpty{List,Seq,Vector,Chain}
- Update sbt-mdoc to 2.3.1 (#584)
- Mdoc 2.3.1, remove no longer necessary hacks (#583)
- Set correct port in docs (#582)
- Update colibri, colibri-jsdom, ... to 0.2.6 (#585)
- fix error reporting in OutwatchTracing
- Fix ScalaJS bundler cache
- add tests for patch events in and outside of observable
- Update sbt-mdoc to 2.3.2 (#589)
- update to cats-effect 3
- make bench compile again
- update colibri to 0.4.0
- improve implicit resolution for render
- drop scala 2.12.15
- update scala to 2.13.8 and deprecate Http util
- deprecated util module: websocket, http, store, localstorage
- remove localstorage mock in tests, works with jsdom natively
- update to colibri 0.4.1
- rename OutWatch to Outwatch and VDomModifier to VModifier
- remove unneeded intermediate instantiation for styles
- simplify domtypes integration types
- use correct document event props for isKeyDown
- little refactoring domtypes
- proxyElement helper inlining
- deprecate emitter(source) for EmitterBuilder.fromSource(source)
- refactor outwatch methods
- small childcommand improvement
- move managed and attr/prop/style to VDomModifier and html/svg to VNode
- emitterbuilder refactoring
- render instance for attribute and emitterbuilder
- increase sbt memory in gh action
- better subscribables in vnode
- delete unused files
- use sbt-tpolecat, remove silencer
- replace remaining silencer with nowarn
- fix warning in jsdocs
- fix deprecations in docs readme
- no fatal warnings in docs
- fix typo in docs
- update docs
- disable tpolecat plugin on docs project
- Update colibri-jsdom to 0.4.2
- Update sbt-tpolecat to 0.2.2 (#600)
- Update README.md
- Update readme.md
- Update README.md
- Use IOApp in readme example
- render instances for cats.effect.Resource
- Revert "Update readme.md"
- update colibri version in docs
- rename OutWatch to outwatch in build.sbt
- add async functions, with functions and asHead/withHead to EmitterBuilder
- fix as and asDelay in EmitterBuilder
- rename remaining OutWatch to Outwatch
- optimization: ignore empty subscriptions
- cleanup, more final modifiers, revise inline
- Update colibri-jsdom to 0.4.5
- improve empty subscription handling
- simplify modifierToSnabbdom subscribables again
- introduce async rendering/patching in micro task
- Remove Scala 2.12 from mergify.yml
- Update sbt-tpolecat to 0.2.3 (#609)
- fix jsdom setup in bench project
- update colibri
- update colibri version in docs
- add scala-java-time dependency to docs for zio
- use macrotaskexecutor in readme docs
- fix wrong deprecated message
- add apply/append/prepend to VModifier
- remove optional attribute builder methods in favor of overloads
- fix readme docs
- Update sbt-tpolecat to 0.3.0 (#613)
- add link to simerplaha/html-to-scala-converter
- Update sbt-tpolecat to 0.3.1 (#614)
- Update scalatest to 3.2.12 (#615)
- Update sbt-scalajs, scalajs-compiler, ... to 1.10.0 (#606)
- Update scalajs-dom to 2.2.0 (#616)
- Update README.md
- Update batches in readme
- Scala 3 (#549)
- Use Scalafmt (#617)
- Use @unchecked to avoid asInstanceOf (#619)
- wip
